back/src/Model/GameRoom.ts

```diff
@@ -206,7 +206,11 @@ export class GameRoom {
     }
 
     updatePlayerDetails(user: User, playerDetailsMessage: SetPlayerDetailsMessage) {
+        const isSilent = user.silent;
         user.updateDetails(playerDetailsMessage);
+        if (!isSilent && user.group !== undefined && user.silent) {
```

Comment:
you check `!userSlient` and `user.silent` so it's weird and I think that you're never inside of this `if`?

Or this parameter can be update in `user.updateDetails(playerDetailsMessage);` and in this case you must to have better code because we could be lost?

Reply:
You're right, it is not easily readable. I will make it cleaner.